vue-components/src/components/LookupMenu.vue

```diff
@@ -47,8 +51,26 @@ export default Vue.extend( {
 				rootElem.style.maxHeight = menuHeight + 'px';
 			}
 		},
-	},
+		onScroll: debounce( function ( this: Vue ) {
+			const rootElem = this.$refs[ 'lookup-menu' ] as HTMLElement;
+			const menuItems = rootElem.querySelectorAll( '.wikit-LookupMenu__item' ) as NodeListOf<HTMLElement>;
```

Comment:
This gives this error: Conversion of type 'Vue | Element | Vue[] | Element[]' to type 'NodeListOf<HTMLElement>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'item' is missing in type 'Element[]' but required in type 'NodeListOf<HTMLElement>'.Vetur(2352)

Reply:
It seems to work when changing the type.